sklearn/compose/_column_transformer.py

```diff
@@ -1049,6 +1055,13 @@ def make_column_transformer(
     )
 
 
+@validate_params(
+    {
+        "pattern": [str, None],
+        "dtype_include": "no_validation",  # delegate to pandas
+        "dtype_exclude": "no_validation",  # delegate to pandas
+    }
+)
 class make_column_selector:
```

Comment:
I did not see it before but `make_column_selector` is a class so we cannot use `validate_params`. Since it is not an estimator (it does not implement `fit`), we cannot currently use the parameter validation framework.